.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ on:
 jobs:
   call-boost-ci:
     name: Run Boost.CI
-    uses: boostorg/boost-ci/.github/workflows/reusable.yml@master
+    uses: boostorg/boost-ci/.github/workflows/reusable.yml@Flamefire-patch-1
```

Comment:
Careful not to check this in.

Reply:
I put the MR on draft for that. Commit removed now that all had passed.